parquet.sublime-settings

```diff
@@ -0,0 +1,4 @@
+{
+	// set to one of 'java' or 'python', depending on which parquet-tools you're using
+	"parquet-tools": "python",
```

Comment:
I went back and forth on default behavior, since this is breaking for existing users (hence the reason for starting with the fork approach) - this could be handled with a release note with the package update, or it could be flipped around with a note for new users.

If/when I or someone else tries a pyarrow/fastparquet approach, we'll probably have a similar issue.